Livecheckables/grok.rb

```diff
@@ -0,0 +1,3 @@
+class Grok
+  livecheck :regex => /^v?(\d+\.[\d]{,3}(\.\d+)+)$/
```

Comment:
```suggestion
  livecheck :regex => /^v?(\d+\.\d{,3}(\.\d+)+)$/
```

The square brackets aren't necessary around `[\d]` here, as `\d{,3}` does the same thing (i.e., square brackets basically read as "one of...", so it's only useful when there's more than one option).